theano/tensor/opt.py

```diff
@@ -1640,6 +1640,8 @@ def local_fill_sink(node):
             if not r:
                 continue
             replacements.update(r)
+    for v in replacements.values():
+        v.tag.values_eq_approx = values_eq_approx_remove_nan
```

Comment:
I didn't understand well, but all those optimizations are removing nan and so without this, it make the daily buildbot fail.